### ISSUE TITLE
Issue reporting fails silently when bugs url is invalid

### ISSUE THREAD

**Issue by digitarald** [microsoft/vscode#175278]
Type: <b>Bug</b>

GitHub Copilot uses a GitHub discussions URL for their `bugs` `url`.

VS Code's Issue Reporter will offer to file an issue for Copilot but then, 🐛, silently fail when clicking `Create on GitHub`.

Could the reporter detect URLs that are not repos and fall back to showing the URL instead?

VS Code version: Code - Insiders 1.76.0-insider (d0acdbff1f30eac7cc68930013485276d1bae0ec, 2023-02-23T05:23:14.565Z)
OS version: Darwin x64 22.3.0
Modes:
Sandboxed: Yes

**Comment by TylerLeonhardt** [comment]
Yeah we should fix this... but I'm unsure if I have same sort of query parameter magic like I do for issues. I'll have to investigate that.
